docker/Dockerfile.sidecar_initializer

```diff
@@ -0,0 +1,3 @@
+FROM scratch
+ADD sidecar-initializer /usr/local/bin/
+ENTRYPOINT ["/usr/local/bin/sidecar-initializer"]
```

Comment:
The current version only ever interacts with the k8s apiserver via patch/list/watch. It uses the in-cluster config similar to pilot discovery service.